.github/workflows/unittests.yml

```
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.6', '3.7', '3.8', '3.9']
+        python-version: ['3.6', '3.7', '3.8', '3.9', "pypy3"]
```

For consistency you might consider using single quotes:

```suggestion
        python-version: ['3.6', '3.7', '3.8', '3.9', 'pypy3']
```